src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -2169,7 +2164,7 @@ public void Overriding_25()
   .method public hidebysig specialname rtspecialname 
           instance void  .ctor() cil managed
   {
-    // Code size       8 (0x8)
+                // Code size       8 (0x8)
```

Comment:
This looks like an unexpected formatting change. There a several similar changes in this file.